homeassistant/components/airvisual/config_flow.py

```diff
@@ -107,33 +109,35 @@ async def async_step_geography(self, user_input=None):
             ):
                 return self.async_abort(reason="already_configured")
 
+        self._check_key_error_step = "geography"
+        self._check_key_error_schema = self.geography_schema
+        return await self.async_step_geography_finish(user_input)
```

Comment:
Why store these on `self` instead of passing them on to `async_step_geography_finish` ?